third_party/gtest-1.8.0/include/gtest/internal/gtest-port.h

```diff
@@ -478,7 +478,7 @@ struct _RTL_CRITICAL_SECTION;
 // detecting whether they are enabled or not.  Therefore, we assume that
 // they are enabled unless the user tells us otherwise.
 #  define GTEST_HAS_EXCEPTIONS 1
-# elif defined(__IBMCPP__) && __EXCEPTIONS
+# elif (defined(__IBMCPP__) || defined(__open_xl__) && defined(__cplusplus)) && __EXCEPTIONS
```

Comment:
for easier readability, can brackets be added to highlight operator precedence?

Reply:
Done

Reply:
please apply/extend to other ifdefs, which were modified in this PR.

Reply:
Added brackets to highlight precedence wherever needed.